src/components/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+export const Header = () => {
+  return (
```

Comment:
when you don't have any logic inside components and just return jsx then you can delete the word `return`

and just return jsx in round brackets

Reply:
ok. I do that cuz make a template for all components and copy it